package.json

```diff
@@ -67,7 +67,7 @@
 		"build": "npm-run-all clean lint transpile",
 		"start": "node scripts/bin/public/index",
 		"start-internal": "node scripts/bin/internal/index",
-		"create-db": "node scripts/bin/create",
+		"create-db": "node scripts/bin/db_data",
```

Comment:
if you want to rename it and this script does a create job then the new name doesn't reflect it, why not `create_db`?